Track Buddy/MotionManager.swift

```diff
@@ -74,32 +91,52 @@ class MotionManager: ObservableObject {
         }
     }
     
-    // TODO: a good place to use @MainActor here
+    private func setUpThrottledPointPublisher() {
+        let cancellable = pointPublisher
+            .throttle(for: Parameters.graphUpdateInterval, scheduler: RunLoop.main, latest: true)
+            .receive(on: DispatchQueue.main, options: nil)
+            .sink(receiveValue: { [weak self] point in
+                self?.throttledPoint = point
+                self?.addToDeque(with: point)
+            })
```

Comment:
Totally fine as-is, but sinks are a pretty good candidate for trailing closures as the usage is common enough for clarity.

```
.sink { ...
}
```